### DISCUSSION TITLE
IC Compilation Error - Multiple Registers Connected to an Input

### DISCUSSION THREAD

**Discussion by stachurr** [MrTJP/ProjectRed#1844] [Q&A] [answered]
I read in a similar discussion post that in-world circuits may not compile in ICs due to a strict rule set. Based on that post & answer, I've explicitly excluded back-propagation and multiple sources driving a single output yet I'm still unable to compile.

Any help/information is greatly appreciated. Here is the blueprint I'm trying to compile (3-bit ripple-carry adder).
Thank you!

![Screenshot 2024-04-08 232109](https://github.com/MrTJP/ProjectRed/assets/25037201/3a839a13-5301-4252-95a0-554826dbbf00)

**Comment by MrTJP** [comment, marked as answer]
Hi. I labeled your image with the issues. You can see that there are multiple gates driving a single input of another gate. You have to explicitly OR together these signals. Let me know if you need more clarification. 

![image](https://github.com/MrTJP/ProjectRed/assets/1957405/b0b1d63c-f5bb-46cd-87d8-0f6c3363c739)

**Reply by stachurr** [reply]
Ohhhhhh I see, thank you kind human being
